scripts/common

```diff
@@ -4,24 +4,35 @@ log (){
 export -f log
 
 bootstrap(){
-	local BOOTSTRAP_CMD=debootstrap
 	local BOOTSTRAP_ARGS=()
 
 	export http_proxy=${APT_PROXY}
 
-	if [ "$(dpkg --print-architecture)" !=  "arm64" ]; then
-		BOOTSTRAP_CMD=qemu-debootstrap
-	fi
-
 	BOOTSTRAP_ARGS+=(--arch arm64)
 	BOOTSTRAP_ARGS+=(--components "main,contrib,non-free")
 	BOOTSTRAP_ARGS+=(--include gnupg)
 	BOOTSTRAP_ARGS+=(--keyring "${STAGE_DIR}/files/debian.gpg")
+	BOOTSTRAP_ARGS+=(--foreign)
 	BOOTSTRAP_ARGS+=("$@")
 	printf -v BOOTSTRAP_STR '%q ' "${BOOTSTRAP_ARGS[@]}"
 
-	capsh --drop=cap_setfcap -- -c "'${BOOTSTRAP_CMD}' $BOOTSTRAP_STR" || true
-
+	capsh --drop=cap_setfcap -- -c "debootstrap $BOOTSTRAP_STR" || true
+	if [ "$(dpkg --print-architecture)" !=  "arm64" ]; then
```

Comment:
Are you able to talk me through exactly what is happening in this block?

Are we sure this is handling everything that `qemu-debootstrap` would do? Any risk that in future versions this could break if `qemu-debootstrap` takes extra steps that we don't yet have implemented here?